devtools/create_env_cpu.sh

```diff
@@ -35,4 +38,4 @@ pip install torch-geometric
 
 # Install other dependencies
 echo "Installing other dependencies..."
-$COMMAND_PKG env update -f cpu_environment.yml
+$COMMAND_PKG env update -f devtools/cpu_environment.yml
```

Comment:
this assumes that you're running the script from outside the `devtools` folder
but for users of TS-GCN that would like to manually run this script, it won't necessarily be correct
Can you add a `cd devtools` command before, and then run the script as if you're inside this folder?

Reply:
I reverted the change cause it may have an effect on the ARC install and our ARC install assumes that this will fail anyway

Reply:
cause in our ARC install, after we run create_env_cpu.sh, and expecting the env update to fail, we run conda env update -f devtools/cpu_environment.yml